service/src/test/resources/ehr/mapper/immunization/expected-output-observation-statement-no-information.xml

```diff
@@ -6,5 +6,14 @@
             <center value="20100630055900"/>
         </effectiveTime>
         <availabilityTime value="20100630054806" />
+        <pertinentInformation typeCode="PERT">
+            <sequenceNumber value="+1"/>
+            <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                <text>Substance: unknown</text>
```

Comment:
Can you confirm that "Substance: unknown" is the correct output in this case?

Reply:
VaccineCode is a required field